app/javascript/packs/application.js

```
@@ -0,0 +1,10 @@
+// Entry point for the application pack
+import "../application";
+import "../channels";
+
+// Import all components
+const componentRequireContext = require.context("../src/components", true);
+const ReactRailsUJS = require("react_ujs");
+ReactRailsUJS.useContext(componentRequireContext);
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)